app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -189,23 +189,25 @@ def supported_documents
                 include: true
               }
             ]
-            @submission.data_source.state_file1099_gs.each do |form1099g|
+
+            @submission.data_source.direct_file_data.w2s.each do |w2|
               supported_docs << {
-                xml: SubmissionBuilder::Ty2022::States::Ny::Documents::State1099G,
-                pdf: nil,
+                xml: SubmissionBuilder::Shared::ReturnW2,
+                pdf: PdfFiller::NyIt2Pdf,
                 include: true,
-                kwargs: { form1099g: form1099g }
+                kwargs: { w2: w2 }
               }
             end
 
-            @submission.data_source.direct_file_data.w2s.each do |w2|
+            @submission.data_source.state_file1099_gs.each do |form1099g|
               supported_docs << {
-                xml: nil,
-                pdf: PdfFiller::NyIt2Pdf,
+                xml: SubmissionBuilder::Ty2022::States::Ny::Documents::State1099G,
+                pdf: nil,
                 include: true,
-                kwargs: { w2: w2 }
+                kwargs: { form1099g: form1099g }
               }
             end
+
```

Comment:
We could add an expectation on the spec/lib/submission_builder/ty2022/states/ny/individual_return_spec.rb and the arizona one that these are present.

Reply:
just a question out of curiosity: why was the order of the 1099 xml and it2 pdf switched for ny?

Reply:
This is not strictly speaking necessary - I wanted to maintain the same order in the XML as is defined in the XSD (FormsBO.xsd). It makes the data easier to correlate.